### ISSUE TITLE
No focal image in Open store

### ISSUE THREAD

**Issue by toraschi** [smurfy/fahrplan#290]
Hello,
Allthoug there is a notice for a focal version in Ut Openstore the version can't be found on the focal channel. Please push the version to the focal channel. Thanks.

**Comment by GerryT11** [comment]
Yes, it would be great to have a Ubuntu Touch Focal version of Fahrplan. Open-Store shows "2.0.33-4-focal", but it is actually not available to install on Focal.
